test/functional/wallet_labels.py

```diff
@@ -190,6 +190,33 @@ def _run_subtest(self, accounts_api, node):
             # already the receiving address of the label. This is a no-op.
             change_label(node, labels[2].receive_address, labels[2], labels[2], accounts_api)
 
+        self.log.info('Check watchonly labels')
```

Comment:
drop 18974?

Reply:
We could probably convert it to test "legacy" addresses instead though it's probably better to do it in a separate PR with a proper commit message. So yeah, let's just skip it for now.